src/org/labkey/test/tests/issues/IssueAPITest.java

```diff
@@ -0,0 +1,377 @@
+package org.labkey.test.tests.issues;
+
+import org.junit.BeforeClass;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.remoteapi.issues.GetIssueCommand;
+import org.labkey.remoteapi.issues.GetIssueResponse;
+import org.labkey.remoteapi.issues.IssuesCommand;
+import org.labkey.remoteapi.issues.IssueModel;
+import org.labkey.remoteapi.issues.IssueResponse;
+import org.labkey.remoteapi.issues.IssueResponseModel;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.TestFileUtils;
+import org.labkey.test.TestTimeoutException;
+import org.labkey.test.categories.InDevelopment;
+import org.labkey.test.util.APIUserHelper;
+import org.labkey.test.util.ApiPermissionsHelper;
+import org.labkey.test.util.IssuesHelper;
+import org.labkey.test.util.PermissionsHelper;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.labkey.test.util.PasswordUtil.getUsername;
+
+@Category({InDevelopment.class})
```

Comment:
Remove 'InDevelopment' category
```suggestion
@Category({})
```

Reply:
dang, I forget that so often/need facepalm emoji here

Reply:
I'm just going to remove that from the template file. It is a relic of our pre-git days. InDevelopment tests now live in feature branches; they don't need a special category.